Source/Analytics/Concepts/Location.cs

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) The International Federation of Red Cross and Red Crescent Societies. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+using Dolittle.Concepts;
+
+namespace Concepts
+{
+    public class Location : Value<Location>
+    {
+        public static readonly Location NotSet = new Location(double.MinValue, double.MinValue);
```

Comment:
Is this used?

Reply:
No – I can remove it for now, but it is often included in concepts to avoid crashing if a Location is missing.